service/host.go

```diff
@@ -323,3 +323,9 @@ func (uis *UIServer) getHostFromCache(hostID string) (*hostCacheItem, error) {
 
 	return &h, nil
 }
+
+func (uis *UIServer) handleBackendError(message string, statusCode int) func(http.ResponseWriter, *http.Request, error) {
+	return func(w http.ResponseWriter, r *http.Request, err error) {
+		gimlet.WriteTextResponse(w, statusCode, message)
```

Comment:
do we have the infrastructure to do the popflash thing? I suppose not. it'd be cool if there was a way to freeze the IDE until it came back.

Reply:
what do you mean by popflash?

Reply:
the thing that the legacy uses to communicate errors on pages, I think the function is called PopFlash or Flash